templates/pages/environment.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "Environments",
+  "layout": [
+    {
+      "type": "widgets",
+      "content": [
+        {
+          "name": "Deployment Button",
+          "x": 0,
+          "y": 0,
+          "width": 3,
+          "height": 3,
+          "definition": "deploymentButton"
+        },
+        {
+          "name": "Deployments view",
+          "width": 12,
+          "height": 28,
+          "definition": "deploymentsView",
+          "configuration": {
+            "customPollingTime": 10,
+            "mapOpenByDefault": false,
+            "mapHeight": 300,
+            "fieldsToShow": [
+              "status",
+              "name",
+              "blueprintName",
+              "location",
+              "subenvironmentsCount",
+              "subservicesCount"
+            ],
+            "pageSize": 50,
+            "sortColumn": "created_at",
+            "sortAscending": null,
+            "filterId": "csys-environment-filter",
+            "filterByParentDeployment": false
```

I still think we should avoid explicitly defining all the configuration details if they are the same as defaults.
Also, because you listed `fieldsToShow`, you missed recently added `id`, so it is another argument for not specifying anything that is not necessary.

My suggestion for this part is to have just:
```suggestion
            "filterId": "csys-environment-filter"
```